mgplvm/kernels/linear.py

```diff
@@ -64,6 +93,16 @@ def scale_sqr(self) -> Tensor:
     def scale(self) -> Tensor:
         return self._scale.abs()
 
+    @property
+    def ell(self) -> Tensor:
+        return softplus(self._ell)
+
     @property
     def msg(self):
-        return ('scale {:.3f} |').format(self.scale.mean().item())
+        return ('scale {:.3f} | ell {:.3f} |').format(self.scale.mean().item(),
+                                                      self.ell.mean().item())
+
+    @staticmethod
+    def dot(x: Tensor, y: Tensor) -> Tensor:
```

Comment:
we probably don't need to define this method, if it's only really used internally and in `K`?

Reply:
wasn't really sure about this either, happy to not define it